### PR TITLE
Almost fix #1544 image rotation not working sometimes

### LINKED ISSUE #1544: image rotation not working sometimes
[deltachat-log-20200812-091030.txt](https://github.com/deltachat/deltachat-android/files/5062858/deltachat-log-20200812-091030.txt)
[deltachat-log-20200812-091030.txt]

Seen in Android 9 phone with the Google keyboard and Delta Chat 1.12.1 on redmi note 8 in MIUI 11 on the last patch, maybe it are the Android 9  

**How do the bug:** 
- open Delta Chat on landscape mode
- go to chat with a Image(in my case .png)
- put the keyboard in fullscreen likewise you are trying to do a message
- rotate to portrait again

### PR DESCRIPTION
**Pull request by Hocuri**
Almost fix #1544 image rotation not working sometimes.

I did not find any better way to do this. Now it is not crashing anymore but it jumps again.

### CODE REVIEW

File: src/org/thoughtcrime/securesms/components/ScaleStableImageView.java [hunk 1]
```diff
@@ -112,9 +112,12 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
...
             Bitmap original = ((BitmapDrawable) large).getBitmap();
-            Bitmap cropped = Bitmap.createBitmap(original, 0, 0, width, height);
-            Drawable croppedDrawable = new BitmapDrawable(getResources(), cropped);
-            overrideDrawable(croppedDrawable);
+            if (height <= original.getHeight()) {
+                Bitmap cropped = Bitmap.createBitmap(original, 0, 0, width, height);
+                Drawable croppedDrawable = new BitmapDrawable(getResources(), cropped);
+                overrideDrawable(croppedDrawable);
+            } // else we do not know what to do.
+            // At least it does not crash anymore but only jumps a little, see https://github.com/deltachat/deltachat-android/issues/1544.
```
Comment (angelo-fuchs) [inline src/org/thoughtcrime/securesms/components/ScaleStableImageView.java:120]: This comment is not stable over time and I would not use it.

**Comment by Hocuri** [comment]
@angelo-fuchs both done.

File: src/org/thoughtcrime/securesms/components/ScaleStableImageView.java [hunk 2]
```diff
@@ -112,9 +112,11 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
                 large = storedSizes.get(landscapeWidth + "x" + landscapeHeight);
             if (large == null) return; // no baseline. can't work.
             Bitmap original = ((BitmapDrawable) large).getBitmap();
-            Bitmap cropped = Bitmap.createBitmap(original, 0, 0, width, height);
-            Drawable croppedDrawable = new BitmapDrawable(getResources(), cropped);
-            overrideDrawable(croppedDrawable);
+            if (height <= original.getHeight() && width <= original.getWidth()) {
```
Comment (angelo-fuchs) [inline src/org/thoughtcrime/securesms/components/ScaleStableImageView.java:115]: That needs to be an ||

Comment (Hocuri) [reply src/org/thoughtcrime/securesms/components/ScaleStableImageView.java]: Done

**Pull request merged**